data/card_catalogue.json

```diff
@@ -628,87 +670,87 @@
+          "tier": "green",
           "details": {
-            "rules": "Throw an object up to 50m",
+            "rules": "Throw an object up to 50 squares",
             "flavour": "\"I will not rest until every window is smashed. They are an insult to me and my kind\" -Little Jimmy, neighborhood rascal",
             "image": "assets/core/card/illustration/Pitch.png"
           }
         },
         {
           "title": "Hurl",
           "upgrade": "Launch",
-          "tier": "green",
+          "tier": "red",
           "details": {
-            "rules": "Throw an object up to 300m",
+            "rules": "Throw an object as far as you want.",
             "flavour": "I want this thing gone, understand? I want to never see it again.",
             "image": "assets/core/card/illustration/Hurl.png"
           }
         },
         {
           "title": "Launch",
           "upgrade": "",
-          "tier": "red",
+          "tier": "gold",
           "details": {
-            "rules": "Throw an object as far as you want",
+            "rules": "Throw an object as far as you want. It cannot be stopped.",
             "flavour": "I guess in a way, we're all still chasing it.",
             "image": "assets/core/card/illustration/Launch.png"
           }
         },
         {
           "title": "Flirt",
           "upgrade": "Charm",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "A compliment increases your rapport with an NPC",
+            "rules": "Increases your rapport with a visible character",
             "flavour": "Are you related to Jean-Claude Van Damme? Because Jean-Claude Van Damme you’re sexy!",
             "image": "assets/core/card/illustration/Flirt.png"
           }
         },
         {
           "title": "Charm",
           "upgrade": "Arouse",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Causes an NPC to like you.",
+            "rules": "Causes a visible character to like you.",
             "flavour": "Do you like raisins? How do you feel about a date?",
             "image": "assets/core/card/illustration/Charm.png"
           }
         },
         {
           "title": "Arouse",
           "upgrade": "Seduce",
-          "tier": "green",
+          "tier": "red",
           "details": {
-            "rules": "Causes an NPC to become attracted to you.",
+            "rules": "Causes a visible character to become attracted to you.",
             "flavour": "If you were a Transformer… you’d be Optimus Fine.",
             "image": "assets/core/card/illustration/Arouse.png"
           }
         },
         {
           "title": "Seduce",
           "upgrade": "",
-          "tier": "red",
+          "tier": "gold",
           "details": {
-            "rules": "Forces an NPC to act on their desire for you.",
+            "rules": "Force any character you can name to act on their desire for you.",
```

Comment:
I'm thinking that you could target someone far away and get zany results as they expend resources to go find you